hydrus/parser/openapi_parser.py

```diff
@@ -224,11 +284,17 @@ def get_paths(doc: Dict["str", Any], classAndClassDefinition: Dict["str",HydraCl
                             api_doc.add_supported_class(
                                 classAndClassDefinition[class_name], collection=True)
                         else:
-                            api_doc.add_supported_class(classAndClassDefinition[class_name], collection=False)
+                            api_doc.add_supported_class(
+                                classAndClassDefinition[class_name], collection=False)
     generateEntrypoint(api_doc)
 
 
-def parse(doc):
+def parse(doc: Dict[str, Any]) -> str:
+    """
+    parent function for parsing the open api documentation
```

Comment:
@vaibhavchellani  You should also use proper punctuations for function explanations in Docstrings.